### ISSUE TITLE
Reading nodal velocity data from RST file fails

### ISSUE THREAD

**Issue by JackGuyver** [pyansys/pymapdl-reader#64]
Hello everybody,

I encounter another problem when reading solution data from an `rst` file that might be related to the already solver issue #59.

I have created and uploaded the minimum example results file [cube.rst.zip](https://github.com/pyansys/pymapdl-reader/files/6944859/cube.rst.zip). My original script uses the `pymapdl-reader` to obtain various nodal data from the results file.
For displacements (using `nodal_time_history(solution_type='NSL')`) and other data this works as expected. However, using the following basic example script:
```
from ansys.mapdl import reader
rstData = reader.read_binary('cube.rst')
nodeIDs, nodalDataAll = rstData.nodal_time_history(solution_type='VEL')
```
raises the following error message:
```
raceback (most recent call last):
  File "./reader_mwe.py", line 10, in <module>
    nodeIDs, nodalDataAll = rstData.nodal_time_history(solution_type='VEL')
  File "TRUNCATED_PATH/ansys-mapdl-reader/0.51.4/python3.8/ansys/mapdl/reader/rst.py", line 1089, in nodal_time_history
    nnum, sol = func(0)
  File "TRUNCATED_PATH/ansys-mapdl-reader/0.51.4/python3.8/ansys/mapdl/reader/rst.py", line 1194, in nodal_velocity
    return self._nodal_solution_result(rnum, 'VSL', in_nodal_coord_sys)
  File "TRUNCATED_PATH/ansys-mapdl-reader/0.51.4/python3.8/ansys/mapdl/reader/rst.py", line 1302, in _nodal_solution_result
    result = result.reshape(-1, sumdof)
ValueError: cannot reshape array of size 243 into shape (4)
```
I have used the latest version `0.51.4` of the reader.

As far as I can tell the velocity values read from the file are not available on all nodes and cartesian components even seem to be partially missing on some nodes (thus the shape that is not a multiple of four).

As mentioned earlier in #59, this result file is the output of a quite uncommon setup, where we try to combine different groups of elements (currently `SURF152`, `SURF154` and `SOLID226`) to perform a simulation of a force and heatflux/temperature loaded structure within a single FEM model.

Is it possible that there is a general problem with this approach?
Since the solver is well capable to handle the model itself and results look reasonable, I assume there is not, but we probably 'simply' cause a lot of trouble by acting outside of the box of intentional use.

Best regards,
  Sebastian

**Comment by akaszynski** [comment]
Sorry @JackGuyver regarding the late reply.

I'll have to fix this regarding the assembly of the results.  Look for an update in the next day or two.

**Comment by akaszynski** [comment]
Will be fixed in #65 and released in `ansys-mapdl-reader==0.51.5`.

```py
from ansys.mapdl import reader as pymapdl_reader
import pyvista
pyvista.set_plot_theme('document')
pyvista.global_theme.cmap = 'jet'

path = '/home/alex/Downloads/cube/cube.rst'
reader = pymapdl_reader.read_binary(path)

node_ids, nodal_data_all = reader.nodal_time_history(solution_type='VEL')

reader.grid.plot(scalars=nodal_data_all[0, :, 0], text='Transient Velocity')
```
![sc](https://user-images.githubusercontent.com/11981631/129084035-467a2295-aec1-4cea-acdd-5859c31ece13.png)

Please let me know if the solution works on your end.

**Comment by JackGuyver** [comment]
`Sorry @JackGuyver regarding the late reply.`
No need to apologize, your reply came, again, much faster than I could expect it.
However, my answer took quite some time since I was out of office due to a combination of holiday and illness. I'm truly sorry for that.

`Please let me know if the solution works on your end.`
It perfectly does. Using `v0.51.5` the velocity is now being read successfully and its distribution also looks correct for my testcase. Thanks a lot for the fix and the version update.
